script/it/caches.txt

```diff
@@ -0,0 +1,190 @@
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory9.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp 5,":a/10,0,3;graphics\system\wnd\cursor1.png",0,0
+	lsp s0_3,kakera_memory1_src,0,0
+		*kak_1i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"Kyaa{p:0:♪} Sakutaroo! It's been so looong!"{n}"You look as deli—I mean—cute as ever~!!"{n}"I'm fi~rst!! Kyaa, he's so fluffy, squishy, and cuuu~te!"`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory1.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory2_src,0,0
+		*kak_2i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"Oooooooooooooohh, come ooooooon!! Whoever wants to eat these bullets Maria chambered for me, step forwaaaaaaard, uwoooo{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}o{0}oooohhh!!!"`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory2.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory3_src,0,0
+		*kak_3i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"............This isn't one of the Master's rules, ...and it certainly isn't one of yours. .........I made this rule myself. .........I'm not furniture anymore. .........I'm the zero on your roulette...!"`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory3.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory4_src,0,0
+		*kak_4i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"Genji‐san. ......Father is calling for you. Keep him company."{n}".........Certainly."`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory4.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory5_src,0,0
+		*kak_5i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"Thank you for understanding, Battler‐kun. ...But you're lucky, you know."{n}"Huh? Why?"{n}"I'd want to leave if I could."{n}"...Ihihihi! I bet you would. I'll let you adults enjoy your happy family chat about the inheritance."`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory5.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory8_src,0,0
+		*kak_8i
+		d2 [ak][text_fade_t 0][text_speed_t 10]`"Alright, Milady... Let's follow that wall and keep walking. ...On and on. And on."`[#][*][\]
+		wait_on_d 0
+		getcursorpos %x,%y
+		amsp 5,%x,%y-33
+		print 1
+		getscreenshot 1920,1080
+		savescreenshot "caches\kakera_memory8.png"
+		d_continue
+		wait_on_d -1
+	textoff
+	lsp s0_3,kakera_memory10_src,0,0
+	if localisation != "it" jumpf
```

Comment:
Why was that added? I do not see it in English.

Reply:
Yeah, I'll immediately cut line 171. Most probably an oversight.